packages/@ember/debug/lib/deprecate.ts

```diff
@@ -170,13 +184,15 @@ if (DEBUG) {
     @param {String} message A description of the deprecation.
     @param {Boolean} test A boolean. If falsy, the deprecation will be displayed.
     @param {Object} options
-    @param {String} options.id A unique id for this deprecation. The id can be
+    @param {String} [options.id] A unique id for this deprecation. The id can be
       used by Ember debugging tools to change the behavior (raise, log or silence)
       for that specific deprecation. The id should be namespaced by dots, e.g.
       "view.helper.select".
-    @param {string} options.until The version of Ember when this deprecation
+    @param {string} [options.until] The version of Ember when this deprecation
       warning will be removed.
     @param {String} [options.url] An optional url to the transition guide on the
+    @param {String} [options.for] A namespace for the deprecation, usually the package name
+    @param {Object} [options.since] Describes when the deprecation became available and enabled.
       emberjs.com website.
```

Comment:
The trailing "emberjs.com website" bit went with `options.url`, and `for`/`since` are not optional.

```suggestion
    @param {String} options.for A namespace for the deprecation, usually the package name
    @param {Object} options.since Describes when the deprecation became available and enabled.
    @param {String} [options.url] An optional url to the transition guide on the
          emberjs.com website.
```

Reply:
ugh, so sloppy, sorry.